### PR TITLE
Sus 428

### PR DESCRIPTION
**Pull request by Archbold**

### CODE REVIEW

File: src/test/java/com/wikia/webdriver/testcases/specialpagestests/FilePageTests.java [hunk 1]
```diff
@@ -133,19 +133,21 @@ public void filePage005_deleteFromHistory() {
     // Load the file page again, should have the same name
     filePage.open(video.getWikiFileName()).verifyEmbeddedVideoIsPresent();
 
-    // Go to the history tab and verify there are at least two videos
-    filePage.selectHistoryTab();
-    filePage.verifyVersionCountAtLeast(2);
+    //Removed following lines until SUS-428 is fixed
+    //// Go to the history tab and verify there are at least two videos
+   //filePage.selectHistoryTab();
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/testcases/specialpagestests/FilePageTests.java]: MAJOR This block of commented-out lines of code should be removed. [rule](http://sonar-s1/coding_rules#rule_key=squid%3ACommentedOutCodeLine)

File: src/test/java/com/wikia/webdriver/testcases/specialpagestests/FilePageTests.java [hunk 2]
```diff
@@ -133,19 +133,21 @@ public void filePage005_deleteFromHistory() {
...
+    //filePage.verifyVersionCountAtLeast(2);
 
-    // Delete the second version
-    DeletePageObject deletePage = filePage.deleteVersion(2);
-    deletePage.submitDeletion();
+    //// Delete the second version
+    //DeletePageObject deletePage = filePage.deleteVersion(2);
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/testcases/specialpagestests/FilePageTests.java]: MAJOR This block of commented-out lines of code should be removed. [rule](http://sonar-s1/coding_rules#rule_key=squid%3ACommentedOutCodeLine)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 2 issues:
* MAJOR 2 major

Watch the comments in this conversation to review them.

**Comment by nikodamn** [comment]
LGTM

File: src/test/java/com/wikia/webdriver/testcases/specialpagestests/FilePageTests.java (hunk 1 above)
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/testcases/specialpagestests/FilePageTests.java]: MAJOR This block of commented-out lines of code should be removed. [rule](http://sonar-s1/coding_rules#rule_key=squid%3ACommentedOutCodeLine)

File: src/test/java/com/wikia/webdriver/testcases/specialpagestests/FilePageTests.java (hunk 2 above)
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/testcases/specialpagestests/FilePageTests.java]: MAJOR This block of commented-out lines of code should be removed. [rule](http://sonar-s1/coding_rules#rule_key=squid%3ACommentedOutCodeLine)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 2 issues:
* MAJOR 2 major

Watch the comments in this conversation to review them.

**Comment by WikiaTech** [comment]
SonarQube analysis reported 2 issues:
* MAJOR 2 major

Watch the comments in this conversation to review them.

**Pull request merged**